docs/pages/usage.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Usage'
+weight: 20
+---
+
+Glee as a spec-driven framework, makes it easier to work with applications in generating documentation and running the development server.
+
+By following the below json config from `package.json`, let's understand the usage of glee defined under `scripts` object.
```

Comment:
I don't think we need this anymore, now that we have a getting-started guide.

Reply:
sorry Souvik. this part explains Glee usages like what glee docs does etc. where as getting started guide explains how it works and skips this part. LMK

Reply:
But as of now it only talks about the Glee CLI commands, Will there be more things added to this doc, if not then you can move this to the installation and talk about how to manually setup glee and then talk about these glee cli commands.

Reply:
I just moved it. thanks for that